docs/topics/forms/formsets.txt

```diff
@@ -22,7 +22,7 @@ a formset out of an ``ArticleForm`` you would do::
     >>> ArticleFormSet = formset_factory(ArticleForm)
 
 You now have created a formset class named ``ArticleFormSet``.
-Instantiating the formset gives you the ability to iterate over the forms 
+Instantiating the formset gives you the ability to iterate over the forms
```

Comment:
I'm so sorry. I've spent a fair amount of time trying to revert this space change and have failed. I don't seem to be able to get git to pick up adding back the trailing space. 😢

Reply:
Let's just leave it.